llvm/llvm.spec.in

```diff
@@ -5,21 +5,20 @@ Summary: LLVM (An Optimizing Compiler Infrastructure)
 License: Apache-2.0 with LLVM exception
 Vendor: None (open source)
 Group: Development/Compilers
-URL: http://llvm..org/
-Source: http://llvm.org/releases/@PACKAGE_VERSION@/@PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz
+URL: https://llvm.org/
```

Comment:
https://llvm.org would do (without the `/` at end)

Reply:
Addressed